File: vue/src/store/entities/record.ts
import Entity from './entity'
import Customer from './customer'


export default class Record extends Entity<number>{
    state?:string
    date?: string
    content?: string
    customerId: number
    salesmanId: number
    salesmanName?: string
    tenantId?: number
    lastModificationTime?: string
    lastModifierUserId?: number
    creationTime?: string
    creatorUserId?: number
    student?:Customer
}

File: aspnet-core/src/EducationAdmin.Core/Sales/Record.cs
using Abp.Domain.Entities;
using Abp.Domain.Entities.Auditing;
using EducationAdmin.Authorization.Users;
using EducationAdmin.Students;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace EducationAdmin.Sales
{
    public class Record : FullAuditedEntity<long>, IMustHaveTenant
    {

        public string State { get; set; }

        public DateTime? Date { get; set; }

        public string Content { get; set; }

        public long CustomerId { get; set; }

        [ForeignKey(nameof(CustomerId))]
        public Customer Customer { get; set; }

        public long SalesmanId { get; set; }
        [ForeignKey(nameof(SalesmanId))]
        public User Salesman { get; set; }
        public int TenantId { get; set; }

    }
}
